diff --git a/src/signal_protocol.c b/src/signal_protocol.c
index c2fdf3d..4cf02a5 100644
--- a/src/signal_protocol.c
+++ b/src/signal_protocol.c
@@ -11,7 +11,7 @@
 #include "signal_utarray.h"
 
 #ifdef _WIN32
-#include "Windows.h"
+#include "windows.h"
 #endif
 
 #ifdef DEBUG_REFCOUNT
diff --git a/src/signal_protocol_types.h b/src/signal_protocol_types.h
index 7e3ab6a..347e563 100644
--- a/src/signal_protocol_types.h
+++ b/src/signal_protocol_types.h
@@ -8,8 +8,10 @@
 #include <unistd.h>
 #else
 #include <basetsd.h>
+#ifndef __ssize_t_defined
 typedef SSIZE_T ssize_t;
 #endif
+#endif
 
 #ifdef __cplusplus
 extern "C" {
