diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3fc4ec6..1e09260 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@
 #
 
 project(usrsctplib C)
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.10)
 
 # Debug build type as default
 if (NOT CMAKE_BUILD_TYPE)
diff --git a/usrsctplib/netinet/sctp_output.c b/usrsctplib/netinet/sctp_output.c
index b8a7b46..819f975 100755
--- a/usrsctplib/netinet/sctp_output.c
+++ b/usrsctplib/netinet/sctp_output.c
@@ -13562,10 +13562,10 @@ sctp_lower_sosend(struct socket *so,
 #endif
 	struct timeval now;
 	struct sctp_block_entry be;
-	struct sctp_inpcb *inp;
+	struct sctp_inpcb *inp = NULL;
 	struct sctp_tcb *stcb = NULL;
 	struct sctp_nets *net;
-	struct sctp_association *asoc;
+	struct sctp_association *asoc = NULL;
 	struct sctp_inpcb *t_inp;
 	struct sctp_nonpad_sndrcvinfo *sndrcvninfo;
 	ssize_t sndlen = 0, max_len, local_add_more;
